pennylane/operation.py

```diff
@@ -1479,6 +1483,10 @@ def inv(self):
         Returns:
             :class:`Operator`: operation to be inverted
         """
+        warnings.warn(
+            "In-place inversion with inv is deprecated. Please use qml.adjoint instead.",
+            UserWarning,
+        )
```

Comment:
Is this required here? We seem to be using the setter in line 1490, doesn't that statement already emit the warning?

Reply:
Good point. Removed the one in `inv()`.